src/test/java/picard/util/IntervalListToolsTest.java

```diff
@@ -406,4 +409,63 @@ public void testLargeScatters() throws IOException {
 
         Assert.assertEquals(gather, original);
     }
+
+    @Test(timeOut = 40_000)
+    public void testCombineAbuttingIntervals() throws IOException {
+        // Test the default behavior of UNION, which is to combine abutting and overlapping intervals.
+        //gather
+        final File ilOut = File.createTempFile("IntervalListTools", ".interval_list");
+        ilOut.deleteOnExit();
+        {
```

Comment:
unneeded brackets (maybe this used to be inside a try block?)

Reply:
I was wondering about this, I copied and pasted from another test. Happy to remove! :)